api/middleware/authRequired.js

```diff
@@ -1,7 +1,15 @@
-const OktaJwtVerifier = require('@okta/jwt-verifier');
-const oktaVerifierConfig = require('../../config/okta');
+/**
```

Comment:
Code like this is simple, clean, to the point, and **_self-documenting_**. One of the checks on the list for a pull request is to remove unnecessary comments. Even if this code weren't self-documenting, GitHub lays out a line-by-line comparison of the changes. In this case, there are four lines of unnecessary comments to replace one single import binding.  This is true throughout.